### PR TITLE
change metrics in automl to support higher dim

### PR DESCRIPTION
**Pull request by TheaperDeng**
Use a reshape operation before calling sklearn.mean_squared_error

### CODE REVIEW

**Comment by yushan111** [comment]
You should add higher dim support for all metrics but not only MSE.

**Comment by TheaperDeng** [comment]
> You should add higher dim support for all metrics but not only MSE.

Complete.

**Comment by yushan111** [comment]
legalize => standardize or something else?

**Comment by yushan111** [comment]
LGTM

**Comment by TheaperDeng** [comment]
PR validation **seems** OK:
http://10.239.47.210:18888/job/ZOO-PR-Validation/4955/
